### ISSUE TITLE
Do validations in item lending 

### ISSUE THREAD

**Issue by shyaman** [cepdnaclk/ce-inventory-api#3]
Dashboard routes are handled in this file
[/extensions/content-manager/controllers/ContentManager.js](../blob/master/extensions/content-manager/controllers/ContentManager.js)

**Comment by namila007** [comment]
we can define regex for each attributes from the content builder

**Comment by shyaman** [comment]
Yeah, also we have to validate values before update the db (ie. before lending an item, the available qty should be >0)

**Comment by namila007** [comment]
ah that logic need to be implement in controller level

**Comment by shyaman** [comment]
yes it's the file I have mentioned above.
